stubs/WordPress/class-wp-post.php

```diff
@@ -12,28 +12,143 @@ final class WP_Post {
 	 */
 	public $ID;
 
+	/**
+	 * @var int
+	 */
+	public $post_author = 0;
+
+	/**
+	 * @var string
+	 */
+	public $post_date = '0000-00-00 00:00:00';
+
+	/**
+	 * @var string
+	 */
+	public $post_date_gmt = '0000-00-00 00:00:00';
+
+	/**
+	 * @var string
+	 */
+	public $post_content = '';
+
+	/**
+	 * @var string
+	 */
+	public $post_title = '';
+
+	/**
+	 * @var string
+	 */
+	public $post_excerpt = '';
+
+	/**
+	 * @var string
+	 */
+	public $post_status = 'publish';
+
+	/**
+	 * @var string
+	 */
+	public $comment_status = 'open';
+
+	/**
+	 * @var string
+	 */
+	public $ping_status = 'open';
+
+	/**
+	 * @var string
+	 */
+	public $post_password = '';
+
+	/**
+	 * @var string
+	 */
+	public $post_name = '';
+
+	/**
+	 * @var string
+	 */
+	public $to_ping = '';
+
+	/**
+	 * @var string
+	 */
+	public $pinged = '';
+
+	/**
+	 * @var string
+	 */
+	public $post_modified = '0000-00-00 00:00:00';
+
+	/**
+	 * @var string
+	 */
+	public $post_modified_gmt = '0000-00-00 00:00:00';
+
+	/**
+	 * @var string
+	 */
+	public $post_content_filtered = '';
+
+	/**
+	 * @var int
+	 */
+	public $post_parent = 0;
+
 	/**
 	 * @var string
 	 */
-	public $post_date;
+	public $guid = '';
+
+	/**
+	 * @var int
+	 */
+	public $menu_order = 0;
 
 	/**
 	 * @var string
 	 */
-	public $post_title;
+	public $post_type = 'post';
 
 	/**
 	 * @var string
 	 */
-	public $post_modified;
+	public $post_mime_type = '';
 
 	/**
 	 * @var int
```

Comment:
Docs says it's a string

Reply:
Looking at the wrong property, I meant `comment_count` :) But probably it's the same story as `post_author`, can you please check?

Reply:
Ok, let's keep it an `int` then :)